plugin/src/main/java/net/aufdemrand/denizen/objects/dLocation.java

```diff
@@ -639,24 +639,127 @@ public String getAttribute(Attribute attribute) {
         }
 
         // <--[tag]
-        // @attribute <l@location.above>
+        // @attribute <l@location.above[<#.#>]>
```

Comment:
Please document as `<l@location.above[(<#.#>)]>`, similar for other tags. The `()` to indicate optional.